extensions/types/vscode-mssql.d.ts

```diff
@@ -730,107 +757,16 @@ declare module 'vscode-mssql' {
 		moveNoneItem(projectUri: string, destinationPath: string, path: string): Promise<ResultStatus>;
 	}
 
-	export interface GetCrossPlatformCompatibilityResult extends ResultStatus {
```

Comment:
these were moved farther down the file in the vscode-mssql version of the file, so they're just moved, not deleted